internal/command/result.go

```diff
@@ -18,8 +18,22 @@
 
 package command
 
+import (
+	"github.com/onflow/flow-cli/pkg/flowcli/util"
+)
+
 type Result interface {
 	String() string
 	Oneliner() string
 	JSON() interface{}
 }
+
+// FieldIncluded checks if field should be included in the result by checking if included flags contains field
+func FieldIncluded(included []string, field string) bool {
+	return util.ContainsStringInsensitive(included, field)
```

Comment:
`FieldIncluded` and `FieldExcluded` do the same thing, which is forward to `util.ContainsStringInsensitive(included, field)`. Should these two functions just be inlined?

Reply:
Yes, I agree. Will push an update.